en/ui-system/components/engine/auto-layout.md

```diff
@@ -2,40 +2,53 @@
+In the horizontal layout, you can set the horizontal orientation with `HorizontalDirection`. There are two types of orientation: `LEFT_TO_RIGHT` and `RIGHT_TO_LEFT`. The former will arrange the nodes from left to right according to their display order in __Hierarchy__. The latter will arrange the nodes from right to left according to their display order.
 
 ### Vertical Layout
 
-![vertical](auto-layout/vertical.png)
+The layout and orientation of vertical layout is almost the same as __horizontal layout__, only the orientation is different, so we won't repeat it here.
 
-When `Layout Type` is set as `Vertical`, all the child nodes will automatically be arrayed vertically and the height of the Layout node will be set up according to the sum `Height` of the child nodes.
+### Grid Layout
 
-In the vertical layout type, the Layout component will not modify the position or width properties of the node on the x axis. Child nodes can only be neatly arrayed by adding the Widget and opening the Left or Right alignment mode.
+Layout `Type` set to `Grid` will start the grid layout. The grid layout will determine the starting point of the layout based on the combination of `HorizontalDirection` and `VerticalDirection` within a fixed container size, and the layout direction based on the `StartAxis` property.
 
-## Node Direction
+### Grid Direction
 
-The Layout arrays' child nodes are based on the display order of child nodes in __Hierarchy__ and refers to the array directions set up by the `Vertical Direction` or `Horizontal Direction` properties.
+Layout arranges the child nodes in the order in which they are displayed in the __Hierarchy__, plus the start point and the alignment direction set by the `StartAxis` property.
 
-### Horizontal Direction
+- Start axis
+  - You can set either `HORIZONTAL` or `VERTICAL` orientation. The former will be aligned horizontally, the latter vertically.
+- Start point
+  - The start point is created by combining `HorizontalDirection` and `VerticalDirection`.
+  - Suppose HorizontalDirection is `LEFT_TO_RIGHT` and VerticalDirection is `TOP_TO_BOTTOM`, then the start point is __top left__
+  - Suppose HorizontalDirection is `RIGHT_TO_LEFT` and VerticalDirection is `BOTTOM_TO_TOP`, then the start point is __bottom right__
 
-You can set up two directions: `Left to Right` or `Right to Left`. The former will array the nodes from left to right according to their display order in __Hierarchy__; the later will array the nodes from right to left according to their display order in __Hierarchy__.
+Two examples are given in conjunction with alignment directions：
 
-### Vertical Direction
+- If the currently set HorizontalDirection is `LEFT_TO_RIGHT`, VerticalDirection is `TOP_TO_BOTTOM` and StartAxis is `HORIZONTAL`, it is telling the component to be sorted horizontally starting from the __top left__ of the container (below left).
```

Comment:
If the currently set HorizontalDirection is `LEFT_TO_RIGHT`, VerticalDirection is `TOP_TO_BOTTOM` and StartAxis is `HORIZONTAL`, it is telling the component to be sorted horizontally starting from the __top left__ of the container (below left).
-->
If the `HorizontalDirection` is set to `LEFT_TO_RIGHT`, `VerticalDirection` is `TOP_TO_BOTTOM` and `StartAxis` is `HORIZONTAL`. This tells the component to be sorted horizontally starting from the __top left__ of the container (below left).